src/components/ImageView/index.js

```diff
@@ -205,16 +210,21 @@ class ImageView extends PureComponent {
                     style={[styles.imageViewContainer, styles.overflowHidden]}
                     onLayout={this.onContainerLayoutChanged}
                 >
-                    <Image
-                        source={{uri: this.props.url}}
+                    <FastImage
+                        source={{
+                            uri: this.props.url,
+                            headers: {
+                                'X-Chat-Attachment-Token': lodashGet(this.props.session, 'encryptedAuthToken', ''),
+                            },
```

Comment:
I think it would be nice to at _least_ put this header string in `CONST`, maybe we can even have a util function that is subscribed to `ONYXKEYS.SESSION` and just returns this entire header object? 🤔